gpii/node_modules/flowManager/test/UpdateTests.js

```diff
@@ -84,92 +78,95 @@ gpii.tests.flowManager.update.testSecondUpdateRequest = function (data) {
 
 gpii.tests.flowManager.update.clearActiveSessions = function (activeSessions) {
```

Comment:
These are just placeholders for actual fixtures - the person responsible for the original implementation should have written proper tests for it, which, like lemon-soaked paper napkins, we dream will one day arise